engine/src/main/java/org/entando/entando/aps/system/services/oauth2/IOAuth2TokenDAO.java

```diff
@@ -30,7 +31,13 @@
     public OAuth2AccessToken getAccessToken(final String accessToken);
 
     public void deleteAccessToken(final String accessToken);
+    
+    public void deleteExpiredToken(int expirationTime);
 
-    public void deleteExpiredToken();
+    public OAuth2RefreshToken readRefreshToken(String tokenValue);
```

Comment:
Why "read" instead of "get"

Reply:
This dao is related to the TokenStore (interface defined by Spring Security) and I leaved the same names where it's possible
https://github.com/spring-projects/spring-security-oauth/blob/4d5517c6597502d769e75bff23b587ee86f3e271/spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/TokenStore.java#L44

Reply:
Ok that is fine for me as long as it matches a known pattern